src/embed.ts

```diff
@@ -19,23 +19,29 @@ import type { DocParser } from 'types/parserContext';
 // ----- Types ----- //
 
 const enum EmbedKind {
-	Generic,
-	Spotify,
-	YouTube,
+	Generic = 'Generic',
+	Spotify = 'Spotify',
+	YouTube = 'YouTube',
```

Comment:
Just curious why you decided to migrate these to `string`s, are you using the `string` values somewhere?

Reply:
So the string value appears as one of the 'data' attributes on the div generated by embedWrapper, the numeric version would work I just thought this might make things clearer if you were ever looking at the dom. 

Im happy either way?